### DISCUSSION TITLE
.server folder #8154

### DISCUSSION THREAD

**Discussion by akomm** [remix-run/remix#8292] [General]
https://github.com/remix-run/remix/pull/8154

I can't test it right now, but noticed that it might be undesired within the routes/ folder. Otherwise anything in the route folder that contains /server/ will be pruned from client, even if a file within contains client components only.

**Comment by kiliman** [comment]
It explicitly has to end in `.server` (notice the leading dot)

**Reply by akomm** [reply]
I know. I'm talking about URL, you talk about the folder name behind the URL. Doesn't change anything.

Any URL that ends with server/ at non-root level would be removed. Or not, that is the question. I did not test it because I did not yet switch my project to vite, its planed later on.

**Reply by kiliman** [reply]
If you need a URL path like `/some/route/with/server/in/the/name`, just escape the server part, although I don't think this is necessary.

`some.route.with.[server].in.the.name`

**Reply by akomm** [reply]
Oh, I've never seen the brackets escape syntax in the docs. Must have missed it somehow.

**Reply by kiliman** [reply]
Here you go.

https://remix.run/docs/en/main/file-conventions/routes#escaping-special-characters

**Reply by akomm** [reply]
Thanks :)
